src/components/index/Header.js

```diff
@@ -98,7 +98,7 @@ const Header = () => (
                     <h2 style={{ fontSize: '2.2rem' }}>
                         Eclipse Theia helps you efficiently develop and deliver multi-language Cloud & Desktop IDEs and tools with modern, state-of-the-art web technologies.
                         <br/>
-                        <a href="https://dev.to/svenefftinge/theia-1-0-finally-a-good-browser-ide-3ok0" rel="noopener noreferrer">Learn about the 1.0 Release!</a>
+                        <a href="https://eclipsesource.com/blogs/2022/01/12/eclipse-theia-1-21-release-news-and-noteworthy/" rel="noopener noreferrer">Learn about the latest 1.21 Release!</a>
```

Comment:
I'm trying to think if we should use `latest`, it will become outdated more quickly, do we plan on making release notes often?

Reply:
Valid comment! I plan to extract a sub page with release notes when we have a few more, so updating this link is a temporary solution. (see https://github.com/eclipse-theia/theia-website/issues/251)